packages/react/dom.js

```diff
@@ -67,4 +69,18 @@ exports.render = function(reactElement, context) {
   };
 };
 
+exports.render = function(reactElement, contextOrConfig) {
+  var context;
+  if (typeof contextOrConfig === 'function') {
```

Comment:
I am afraid we are going to need a different check here - `contextOrConfig` could be a context instance here, but I would not really expect a function/constructor.

Reply:
I think we ought to use the brand new [`isMixinable()`](https://www.npmjs.com/package/mixinable#defineismixinableobject) function here - and we could additionally check whether we have a proper `Context`.

Reply:
Yes, was broken, works fine with `isMixinable` :)